<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-4VURPJC" failures="0" tests="2" name="com.actitime1.testscript.CustomerModule" time="19.269" errors="1" timestamp="2023-07-23T21:30:16 IST" skipped="1">
  <testcase classname="com.actitime1.testscript.CustomerModule" name="testCreateCustomer" time="0.001">
    <skipped/>
  </testcase> <!-- testCreateCustomer -->
  <testcase classname="com.actitime1.testscript.CustomerModule" name="openBrowser" time="19.268">
    <error message="Unable to obtain: Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}" type="org.openqa.selenium.remote.NoSuchDriverException">
      <![CDATA[org.openqa.selenium.remote.NoSuchDriverException: Unable to obtain: Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}
at org.openqa.selenium.remote.service.DriverFinder.getPath(DriverFinder.java:29)
at org.openqa.selenium.chrome.ChromeDriver.generateExecutor(ChromeDriver.java:98)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:87)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:82)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:50)
at com.actitime.generic.BaseClass.openBrowser(BaseClass.java:22)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:663)
at org.testng.TestRunner.beforeRun(TestRunner.java:652)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.WebDriverException: Unsuccessful command executed: [C:\Users\Welcome\AppData\Local\Temp\selenium-manager70785205721070017943985075726104914\selenium-manager.exe, --browser, chrome, --output, json]
error sending request for url (https://chromedriver.storage.googleapis.com/LATEST_RELEASE_114): connection error: unexpected end of file
Build info: version: '4.10.0', revision: 'c14d967899'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '19.0.2'
Driver info: driver.version: ChromeDriver
at org.openqa.selenium.manager.SeleniumManager.runCommand(SeleniumManager.java:130)
at org.openqa.selenium.manager.SeleniumManager.getDriverPath(SeleniumManager.java:251)
at org.openqa.selenium.remote.service.DriverFinder.getPath(DriverFinder.java:27)
... 27 more
]]>
    </error>
    <system-out>
      <![CDATA[openBrowser]]>
    </system-out>
  </testcase> <!-- openBrowser -->
  <system-out>
    <![CDATA[openBrowser]]>
  </system-out>
</testsuite> <!-- com.actitime1.testscript.CustomerModule -->
